packages/frontend/src/components/campaigns/pool/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Typography } from "@/src/ui/typography";
+import styles from "./styles.module.css";
```

Comment:
I'd keep the styles as the last import like the other components.

Reply:
Addressed in 1cc81e200b9035b50c9a7c9f8a5721dbae9926bd